raster_loader/cli/bigquery.py

```diff
@@ -24,11 +24,20 @@ def bigquery(args=None):
 @click.option("--project", help="The name of the Google Cloud project.", required=True)
 @click.option("--dataset", help="The name of the dataset.", required=True)
 @click.option("--table", help="The name of the table.", default=None)
-@click.option("--band", help="Band within raster to upload.", default=1)
+@click.option(
```

Comment:
Should we allow repeating or do comma separated?

Reply:
default way to manage muliple parameter via Click module si repeating --band (AFAIK), I preferred to use native method than introducing a parser

Reply:
btw can be changed

Reply:
Ok if it's the default way let's stick to it. I'm too used to the comma separated format.

Reply:
me too, infact I started to implement in this way, but required more times and I decider that solution that allow me to focus on tests and stability.